games/a.yaml

```
@@ -2,12 +2,16 @@
   lang: Java
   license:
   - As-is
-  development: active
+  development: halted
   originals:
   - Rodent's Revenge
   repo: https://github.com/maxxboehme/A_Mouses_Vengeance
   type: remake
-  updated: 2017-04-04
+  updated: 2019-04-16
+  status: playable
+  content: open
```

```suggestion
  content: free
```
I can't find any content license